packages/ion_identity_client/lib/src/auth/dtos/credential_request_data.dart

```diff
@@ -6,12 +6,15 @@ import 'package:json_annotation/json_annotation.dart';
 
 part 'credential_request_data.g.dart';
 
+// ignore: constant_identifier_names
+enum CredentialKind { Fido2, Key, PasswordProtectedKey, RecoveryKey }
+
 @JsonSerializable()
 class CredentialRequestData {
```

Comment:
We use this class for a recovery creds request. It shouldn't be used for register.